restapi/src/main/java/io/stargate/web/impl/Server.java

```diff
@@ -150,8 +150,8 @@ protected void configure() {
 
     // Documents API
     environment.jersey().register(new DocsApiComponentsBinder(environment));
-    environment.jersey().register(DocumentResourceV2.class);
     environment.jersey().register(ReactiveDocumentResourceV2.class);
+    environment.jersey().register(DocumentResourceV2.class);
```

Comment:
Is it possible that the 404's you are observing are due to this reordering? I forget how Dropwizard chooses to resolve endpoints when the path you provide techncially matches multiple paths

Reply:
Yea I added reordering in hope to actually solve it.. Forgot to revert.. Are there any matching ones to this one?

Reply:
Fixed.